### PR TITLE
Pd 86 sanitize objectstore

### PR DESCRIPTION
**Pull request by andreipope**
@harsh-px Sanitized and updated the content.

### CODE REVIEW

**Comment by andreipope** [comment]
@harsh-px kind reminder

File: content/reference/CLI/objectstore.md [hunk 1]
```diff
@@ -1,126 +1,211 @@
 ---
-title: Objectstore on PX Cluster
-keywords: cloud, objectstore
+title: Objectstore on PX Cluster using pxctl
+keywords: cloud, objectstore, pxctl
```
Comment (harsh-px) [inline content/reference/CLI/objectstore.md]: this needs a shorter link title

Comment (andreipope) [reply content/reference/CLI/objectstore.md]: @harsh-px added a shorter linkTitle. Anyway, this page is hidden, so it doesn't show up. Should I unhide it?

Comment (harsh-px) [reply content/reference/CLI/objectstore.md]: No, don't unhide it. It would be hidden for a purpose.

**Pull request merged**